ivy/functional/frontends/torch/miscellaneous_ops.py

```diff
@@ -376,3 +376,10 @@ def atleast_3d(*tensors):
 @to_ivy_arrays_and_back
 def diag(input, diagonal=0, *, out=None):
     return ivy.diag(input, k=diagonal)
+
+
+def corrcoef(input):
+    input = ivy.asarray(input)
+    if input.shape[1] > 2:
+        raise ValueError("m has more than 2 dimensions")

```

Comment:
The message should be like this -: 
`Expected 2D input array (N, M), but got input array of shape ({input.shape[1]}) instead`